api/v1alpha1/component_types.go

```diff
@@ -41,6 +41,11 @@ type GitSource struct {
 	// Optional.
 	Revision string `json:"revision,omitempty"`
 
+	// The commit id (SHA-1 checksum) of the latest component build.
+	// Example: 41fbdb124775323f58fd5ce93c70bb7d79c20650.
+	// Optional.
+	Commit string `json:"commit,omitempty"`
```

Comment:
Should this go in the status instead of the Component spec?

Currently `Revision` can be used to specify the branch, tag, or Commit ID for the source repo, so I feel that Commit would be duplicated?

Reply:
My preference would be status for both.

FWIW, there's also a status field for containerImage: https://github.com/redhat-appstudio/application-api/blob/main/api/v1alpha1/component_types.go#L151

Reply:
That is true, though the status.containerImage represents the last container image that was created and bound to the Development environment.

In any case I agree with moving the new field to status. Since it has a specific use case, I also renamed it to `lastBuiltCommit`.
Please let me know if that looks good.

Reply:
Sounds good to me